src/schema.sql

```diff
@@ -36,30 +52,28 @@ CREATE TABLE bounds (
 CREATE INDEX bounds_gist ON bounds USING GIST(geom);
 CREATE INDEX bounds_idx ON bounds(name);
 
-DROP TABLE IF EXISTS users;
-CREATE TABLE users (
-    id          BIGSERIAL PRIMARY KEY,
-    access      TEXT,
-    username    TEXT UNIQUE NOT NULL,
-    password    TEXT NOT NULL,
-    email       TEXT UNIQUE NOT NULL,
-    meta        JSONB
-);
-
-DROP TABLE IF EXISTS users_tokens;
 CREATE TABLE users_tokens (
     name        TEXT,
-    uid         BIGINT,
+    uid         BIGINT REFERENCES users(id),
     token       TEXT PRIMARY KEY,
     expiry      TIMESTAMP,
     scope       TEXT
 );
 
-DROP TABLE IF EXISTS geo;
-DROP INDEX IF EXISTS geo_gist;
-DROP INDEX IF EXISTS geo_idx;
+CREATE TABLE deltas (
+    id          BIGSERIAL PRIMARY KEY,
+    created     TIMESTAMP,
+    features    JSONB,
+    affected    BIGINT[],
+    props       JSONB,
+    uid         BIGINT REFERENCES users(id),
+    finalized   BOOLEAN DEFAULT FALSE
+);
+CREATE INDEX deltas_idx ON deltas(id);
+CREATE INDEX deltas_affected_idx on deltas USING GIN (affected);
+
 CREATE TABLE geo (
-    id          BIGSERIAL UNIQUE,
+    id          BIGSERIAL PRIMARY KEY,
```

Comment:
`UNIQUE` and `PRIMARY KEY` are pretty similar (identical?), what's the reasoning here?

Reply:
The difference is that `UNIQUE` allows one or more `NULL` values whereas `PRIMARY KEY` enforces both a `UNIQUE` and a `NOT NULL` constraint. I originally added this as it was required by Diesel, but per chat with @ingalls, is probably something we should add anyway (though we don't have to do it in this PR)

Reply:
I'd really prefer to keep a PR like this clean of unrelated changes. If this adds any time to the migration at all, I'm very against it. I totally get the desire to make changes like this, but I'd vote against schema maintenance in here.

Reply:
Amended previous commit to remove this